plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/basic-setup.spec.js

```diff
@@ -1,15 +1,29 @@
 const { test, expect } = require( '@playwright/test' );
+const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 
 test.describe(
 	'Store owner can finish initial store setup',
 	{ tag: [ '@gutenberg', '@payments', '@services' ] },
 	() => {
 		test.use( { storageState: process.env.ADMINSTATE } );
+
+		test.beforeAll( async ( { baseURL } ) => {
+			const api = new wcApi( {
+				url: baseURL,
+				consumerKey: process.env.CONSUMER_KEY,
+				consumerSecret: process.env.CONSUMER_SECRET,
+				version: 'wc/v3',
+			} );
```

Comment:
Instead of creating a new instance of wcApi you can use the existing `api` fixture from [fixtures.js](https://github.com/woocommerce/woocommerce/blob/06b9380ab44557fc47357777fcccf39dc387b287/plugins/woocommerce/tests/e2e-pw/fixtures/fixtures.js#L7)

Reply:
Will be done in a new PR https://github.com/woocommerce/woocommerce/pull/50464